src/pydocstyle/checker.py

```diff
@@ -28,6 +28,24 @@ def decorator(f):
     return decorator
 
 
+
+FSTRING_RE = re(r'^[rR]?[fF]')
```

Comment:
Maybe case insensitive regex will be better?

Reply:
Case insensitive regexes invoke Unicode case conversion rules and could allow in other code points that may not be valid. Possibly `re.IGNORECASE | re.ASCII` might work but that's more verbose than this.